src/momento/internal/_utilities/_grpc_channel_options.py

```diff
@@ -8,11 +8,14 @@
 DEFAULT_MAX_MESSAGE_SIZE = 5_243_000  # bytes
 
 
-def grpc_channel_options_from_grpc_config(grpc_config: GrpcConfiguration) -> grpc.aio.ChannelArgumentType:
+def grpc_channel_options_from_grpc_config(
+    grpc_config: GrpcConfiguration, is_control_client: bool = False
+) -> grpc.aio.ChannelArgumentType:
     """Create gRPC channel options from a GrpcConfiguration.
 
     Args:
         grpc_config (GrpcConfiguration): the gRPC configuration.
+        is_control_client (bool, optional): whether the client is a control client, in which case we want to disable keepalives. Defaults to False.
```

Comment:
nit: for clarity's sake I prefer separate functions, eg:

`grpc_data_channel_options_from_grpc_config`

`grpc_control_channel_options_from_grpc_config`

And it's ok for the implementation of the latter to just invoke the former and override settings where appropriate.